pkgs/development/libraries/quarto/default.nix

```diff
@@ -18,35 +19,33 @@
 
 stdenv.mkDerivation (final: {
   pname = "quarto";
-  version = "1.3.450";
+  version = "1.4.550";
   src = fetchurl {
     url = "https://github.com/quarto-dev/quarto-cli/releases/download/v${final.version}/quarto-${final.version}-linux-amd64.tar.gz";
-    sha256 = "sha256-bcj7SzEGfQxsw9P8WkcLrKurPupzwpgIGtxoE3KVwAU=";
+    sha256 = "sha256-cWHd7ZWGBdRTaSHYVa8LuTDA5gefJ5baOGERS2g6Vvg=";
   };
 
   nativeBuildInputs = [
     makeWrapper
   ];
 
-  patches = [
-    ./fix-deno-path.patch
-  ];
-
   postPatch = ''
     # Compat for Deno >=1.26
     substituteInPlace bin/quarto.js \
       --replace 'Deno.setRaw(stdin.rid, ' 'Deno.stdin.setRaw(' \
-      --replace 'Deno.setRaw(Deno.stdin.rid, ' 'Deno.stdin.setRaw('
+      --replace 'Deno.setRaw(Deno.stdin.rid, ' 'Deno.stdin.setRaw(' \
```

Comment:
We should switch to --replace-fail

Reply:
I'm getting 

```
substituteStream(): ERROR: Invalid command line argument: --replace-fail

```

Reply:
You might want to rebase this your branch to the latest version of master

Reply:
Thanks! But now I get this:

```
substituteStream(): ERROR: pattern Deno.setRaw\(stdin.rid\,\  doesn't match anything in file 'bin/quarto.js'
/nix/store/ysygj1dnlf8zqka5cva7ydi3x8n3h8cd-stdenv-linux/setup: line 131: pop_var_context: head of shell_variables not a function context
```

`replace` works though

Reply:
I think that if `--replace-fail` fails because it doesn't match anything, then that means this line doesn't do anything and can be removed.